recipe/meta.yaml

```
@@ -22,14 +22,14 @@ requirements:
     - pip
     - setuptools_scm
     - numpy >=1.16
```

Let's use the pinning version in `host`
```suggestion
    - numpy
```

---

it seems that removing numpy pinning introduced a conflict 

```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {"numpy[version='1.13.*|1.15.*|>=1.14.6,<1.18.0|>=1.14.6,<2.0a0|>=1.16.5,<2.0a0|>=1.9.3,<2.0a0|>=1.15.4,<2.0a0|>=1.11|>=1.11.3,<2.0a0|>=1.12,<1.13.0a0|>=1.14,<1.15.0a0|>=1.13,<1.14.0a0|>=1.10,<1.11.0a0|>=1.11,<1.12.0a0']", "python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|>=3.8,<3.9.0a0']", "packaging[version='>=20.2']", "python[version='2.7.*|3.5.*|3.6.*|>=3.5,<3.6.0a0|>=3.9,<3.10.0a0|3.4.*']", 'ca-certificates', "libgfortran-ng[version='>=7,<8.0a0']", "python[version='>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']", "libopenblas[version='0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.10|0.3.11|0.3.11|0.3.11|0.3.11|0.3.11|0.3.11|0.3.11|0.3.11|0.3.3|0.3.3|0.3.3|>=0.3.10,<0.3.11.0a0|>=0.3.10,<1.0a0|>=0.3.9,<0.3.10.0a0|>=0.3.9,<1.0a0|>=0.3.8,<0.3.9.0a0|>=0.3.8,<1.0a0|>=0.3.7,<0.3.8.0a0|>=0.3.7,<1.0a0|>=0.3.6,<0.3.7.0a0|>=0.3.6,<1.0a0|>=0.2.20,<0.2.21.0a0|0.3.9|0.3.8|0.3.7|0.3.7|0.3.7|0.3.7|0.3.7|0.3.7|0.3.7|0.3.7|0.3.6|0.3.6|0.3.6|0.3.6|0.3.6|0.3.6|0.3.6',build='h5a2b251_0|h5a2b251_1|h5a2b251_2|h5a2b251_0|h6e990d7_4|h6e990d7_5|h6e990d7_1|h6e990d7_2|h5ec1e0e_4|h5ec1e0e_5|h5ec1e0e_6|h5ec1e0e_0|pthreads_hb3c22a3_1|pthreads_hb3c22a3_3|openmp_h709eae2_3|pthreads_hb3c22a3_4|pthreads_h4812303_5|pthreads_hb3c22a3_5|openmp_h709eae2_5|openmp_h709eae2_0|pthreads_h4812303_0|pthreads_hb3c22a3_0|openmp_h59f9010_0|pthreads_h4812303_1|pthreads_hb3c22a3_1|openmp_h709eae2_1|openmp_h59f9010_1|h5a2b251_2|h5a2b251_3|h5a2b251_1|openmp_h59f9010_5|openmp_h709eae2_4|pthreads_hb3c22a3_2|openmp_h709eae2_2|openmp_h709eae2_1|h5ec1e0e_0|h5ec1e0e_0|h5ec1e0e_7|h6e990d7_3|h6e990d7_0|h6e990d7_6|h6e990d7_3']", "numpy[version='1.10.*|1.11.*|1.12.*|1.13.*|>=1.11|>=1.11.*|>=1.12.1,<2.0a0|>=1.14.6,<2.0a0|>=1.15.4,<2.0a0|>=1.16.5,<2.0a0|>=1.18.5,<2.0a0|>=1.19.2,<2.0a0|>=1.18.4,<2.0a0|>=1.18.1,<2.0a0|>=1.9.3,<2.0a0|>=1.9.*|>=1.9|>=1.8|>=1.7|>=1.16.6,<2.0a0|>=1.13.3,<2.0a0|>=1.11.3,<2.0a0']", "numpy-base==1.11.3[build='py37h2b20989_7|py27h2b20989_7|py36h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_8|py37hdbf6ddf_8|py36h7cdd4dd_9|py37h3dfced4_9|py35h3dfced4_9|py27h3dfced4_9|py37h81de0dd_9|py27h74e8950_9|py35h74e8950_9|py27h74e8950_10|py36h74e8950_10|py27h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py37h2f8d375_11|py27h2f8d375_11|py36hde5b4d6_11|py37hde5b4d6_11|py36h2f8d375_12|py38hde5b4d6_12|py38h2f8d375_12|py36hde5b4d6_12|py37hde5b4d6_12|py27hde5b4d6_12|py27h2f8d375_12|py37h2f8d375_12|py27hde5b4d6_11|py36h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py37h81de0dd_10|py35h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py27h81de0dd_9|py36h74e8950_9|py37h74e8950_9|py36h81de0dd_9|py36h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py27hdbf6ddf_8|py27h2b20989_8|py37h2b20989_8|py36h2b20989_8|py37hdbf6ddf_7|py36hdbf6ddf_7']", "python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']", "openblas[version='0.3.5.*|>=0.3.6,<0.3.7.0a0']", "libgfortran-ng[version='>=4.9|>=8,<9.0a0']", "mkl[version='>=2018.0.3,<2019.0a0|>=2019.1,<2021.0a0|>=2019.4,<2021.0a0']", "libcblas==3.8.0[build='4_mkl|5_mkl|6_mkl|7_mkl|12_mkl|13_mkl|15_mkl|4_openblas|6_openblas|7_openblas|12_openblas|13_openblas|14_openblas|15_openblas|16_openblas|17_openblas|11_openblas|10_openblas|9_openblas|8_openblas|5_openblas|16_mkl|14_mkl|11_mkl|10_mkl|9_mkl|8_mkl']", 'setuptools', "_libgcc_mutex[version='*|0.1',build='conda_forge|main']", "python[version='>=3.9,<3.10.0a0']", 'tzdata', "openblas[version='0.2.18.*|0.2.18|0.2.18.*|0.2.19|0.2.19.*|0.2.20|0.2.20.*|>=0.2.20,<0.2.21.0a0|>=0.3.3,<0.3.4.0a0|>=0.2.15']", "mkl[version='2019.1.*|>=2019.0,<2020.0a0|>=2020.0,<2021.0a0|>=2020.2,<2021.0a0|>=2019.4,<2020.0a0|>=2019.3,<2020.0a0|>=2019.3,<2021.0a0']", "numpy-base[version='1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.11.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.3|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.4|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.5|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.14.6|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.0|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.1|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.2|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.3|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.15.4|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.0|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.1|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.2|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.3|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.4|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.5|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.16.6|1.17.2.*|1.17.3.*|1.17.4.*|1.18.1.*|1.18.5.*|1.19.1|1.17.0|1.17.0|1.17.0|1.17.0',build='py37h2b20989_7|py27h2b20989_7|py36h2b20989_7|py27hdbf6ddf_7|py36hdbf6ddf_7|py37hdbf6ddf_8|py27h2b20989_8|py36h7cdd4dd_9|py37h3dfced4_9|py35h3dfced4_9|py27h3dfced4_9|py36h81de0dd_9|py37h81de0dd_9|py27h74e8950_9|py35h74e8950_9|py37h74e8950_9|py27h81de0dd_9|py36h74e8950_10|py27h81de0dd_10|py37h2f8d375_10|py36h2f8d375_10|py27h2f8d375_11|py36hde5b4d6_11|py37hde5b4d6_11|py27hde5b4d6_12|py36h0ea5e3f_1|py27h0ea5e3f_1|py27h9be14a7_1|py35h9be14a7_1|py27h2b20989_0|py36hdbf6ddf_0|py35hdbf6ddf_0|py27h2b20989_0|py36hdbf6ddf_0|py27hdbf6ddf_0|py35hdbf6ddf_0|py36h2b20989_1|py37hdbf6ddf_1|py27h2b20989_1|py36h2b20989_2|py37h2b20989_2|py37hdbf6ddf_2|py36hdbf6ddf_2|py36h2b20989_3|py37h2b20989_3|py27hdbf6ddf_4|py37hdbf6ddf_4|py36hdbf6ddf_4|py35h2b20989_4|py37h2f8d375_4|py36h2f8d375_4|py27h2f8d375_4|py37h81de0dd_4|py38h2f8d375_4|py38hde5b4d6_4|py36h2f8d375_5|py37h2f8d375_5|py27h2f8d375_5|py37hde5b4d6_5|py36h7cdd4dd_0|py35h7cdd4dd_0|py27h3dfced4_0|py35h3dfced4_0|py35h74e8950_0|py37h74e8950_0|py27h81de0dd_0|py37h81de0dd_0|py35h81de0dd_0|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py27h2f8d375_0|py27h81de0dd_0|py35h81de0dd_0|py35h2f8d375_0|py36h2f8d375_0|py36h81de0dd_1|py37h81de0dd_1|py37h2f8d375_1|py27h2f8d375_0|py27h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py27h2f8d375_0|py37h81de0dd_0|py27h81de0dd_0|py36hde5b4d6_0|py37h2f8d375_0|py36hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_1|py36hde5b4d6_1|py37h2f8d375_0|py27h2f8d375_0|py36h2f8d375_0|py36hde5b4d6_0|py37hde5b4d6_0|py37hde5b4d6_1|py36hde5b4d6_1|py36h2f8d375_0|py27h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py27h2f8d375_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py36h2f8d375_0|py37h2f8d375_0|py37hde5b4d6_0|py37h2f8d375_0|py36h2f8d375_0|py37hde5b4d6_0|py36hde5b4d6_0|py38h2f8d375_0|py38h75fe3a5_0|py38hfa32c7d_0|py37h75fe3a5_0|py36h75fe3a5_0|py36hfa32c7d_0|py37hfa32c7d_0|py27hde5b4d6_0|py27h2f8d375_0|py38hde5b4d6_0|py36hde5b4d6_0|py27hde5b4d6_0|py27h2f8d375_0|py27hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py27h2f8d375_0|py36hde5b4d6_0|py27hde5b4d6_0|py37h2f8d375_0|py27hde5b4d6_1|py36h2f8d375_1|py37h2f8d375_1|py27h2f8d375_1|py27hde5b4d6_0|py27hde5b4d6_1|py36h2f8d375_1|py27h2f8d375_1|py37h2f8d375_1|py27hde5b4d6_0|py37hde5b4d6_0|py27h2f8d375_0|py37hde5b4d6_0|py36h2f8d375_0|py37hde5b4d6_0|py27hde5b4d6_0|py36h81de0dd_0|py37h81de0dd_0|py36h81de0dd_0|py37h2f8d375_0|py36h2f8d375_0|py27h81de0dd_1|py27h2f8d375_1|py36h2f8d375_1|py36h81de0dd_0|py37h81de0dd_0|py37h2f8d375_0|py35h2f8d375_0|py36h81de0dd_0|py36h74e8950_0|py27h74e8950_0|py37h3dfced4_0|py36h3dfced4_0|py37h7cdd4dd_0|py27h7cdd4dd_0|py36hde5b4d6_5|py27hde5b4d6_5|py36h81de0dd_4|py35h81de0dd_4|py27h81de0dd_4|py35h2f8d375_4|py35hdbf6ddf_4|py36h2b20989_4|py37h2b20989_4|py27h2b20989_4|py27hdbf6ddf_3|py27h2b20989_3|py36hdbf6ddf_3|py37hdbf6ddf_3|py27hdbf6ddf_2|py27h2b20989_2|py36hdbf6ddf_1|py27hdbf6ddf_1|py37h2b20989_1|py36h2b20989_0|py27hdbf6ddf_0|py35h2b20989_0|py36h2b20989_0|py36h9be14a7_1|py35h0ea5e3f_1|py38h2f8d375_12|py38hde5b4d6_12|py36hde5b4d6_12|py37hde5b4d6_12|py36h2f8d375_12|py27h2f8d375_12|py37h2f8d375_12|py27hde5b4d6_11|py36h2f8d375_11|py37h2f8d375_11|py35h2f8d375_10|py27h2f8d375_10|py35h81de0dd_10|py36h81de0dd_10|py37h81de0dd_10|py27h74e8950_10|py35h74e8950_10|py37h74e8950_10|py35h81de0dd_9|py36h74e8950_9|py36h3dfced4_9|py37h7cdd4dd_9|py27h7cdd4dd_9|py35h7cdd4dd_9|py35h2b20989_8|py35hdbf6ddf_8|py27hdbf6ddf_8|py37h2b20989_8|py36h2b20989_8|py36hdbf6ddf_8|py37hdbf6ddf_7']", "libcblas[version='>=3.8.0,<4.0a0']", "libopenblas[version='>=0.3.2,<0.3.3.0a0|>=0.3.3,<1.0a0']", "liblapack==3.8.0[build='4_mkl|5_mkl|6_mkl|7_mkl|12_mkl|13_mkl|15_mkl|4_openblas|6_openblas|7_openblas|12_openblas|13_openblas|14_openblas|15_openblas|16_openblas|17_openblas|11_openblas|10_openblas|9_openblas|8_openblas|5_openblas|16_mkl|14_mkl|11_mkl|10_mkl|9_mkl|8_mkl']", "python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|3.4.*|>=3.7,<3.8.0a0']", "python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.8,<3.9.0a0']", 'six', "liblapack[version='>=3.8.0,<4.0.0a0']", "python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|3.8.*|3.7.*']", "python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3|>=3.8,<3.9.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|3.4.*']", "libblas[version='>=3.8.0,<4.0a0']", "numpy[version='>=1.11,<2.0a0|>=1.16,<2.0a0']", "libgfortran[version='>=3.0']", "python[version='>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']", 'libgfortran', 'python', "libblas[version='3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0.*|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.8.0|3.9.0.*',build='4_mkl|5_blis|5_mkl|6_blis|6_openblas|6_mkl|7_mkl|8_blis|8_mkl|9_openblas|10_mkl|11_blis|11_openblas|12_blis|12_openblas|12_mkl|13_mkl|15_mkl|16_openblas|16_blis|16_mkl|17_openblas|15_openblas|15_blis|14_mkl|14_blis|14_openblas|13_blis|13_openblas|11_mkl|10_openblas|10_blis|9_mkl|9_blis|8_openblas|7_openblas|7_blis|5_openblas|4_openblas|4_blis']", "certifi[version='>=2016.09|>=2016.9.26']"}

```
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=228658&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=604


I am going to add the pinning back to check if it is the problem.